server/shared/src/main/scala/org/http4s/server/package.scala

```diff
@@ -43,31 +44,41 @@ package object server {
          |             |_|""".stripMargin.split("\n").toList
 
     val IPv4Host: String =
-      InetAddress.getByAddress("localhost", Array[Byte](127, 0, 0, 1)).getHostAddress
+      if (Platform.isJvm)
+        InetAddress.getByAddress("localhost", Array[Byte](127, 0, 0, 1)).getHostAddress
+      else
+        "127.0.0.1"
     val IPv6Host: String =
-      InetAddress
-        .getByAddress("localhost", Array(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1))
-        .getHostAddress
+      if (Platform.isJvm)
+        InetAddress
+          .getByAddress("localhost", Array(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1))
+          .getHostAddress
+      else "0:0:0:0:0:0:0:1"
 
     @deprecated(
       message =
         "Please use IPv4Host or IPv6Host. This value can change depending on Platform specific settings and can be either the canonical IPv4 or IPv6 address. If you require this behavior please call `InetAddress.getLoopbackAddress` directly.",
       since = "0.21.23",
     )
-    val Host = InetAddress.getLoopbackAddress.getHostAddress
+    def Host = InetAddress.getLoopbackAddress.getHostAddress
     val HttpPort = 8080
 
-    val IPv4SocketAddress: InetSocketAddress =
+    def IPv4SocketAddress: InetSocketAddress =
       InetSocketAddress.createUnresolved(IPv4Host, HttpPort)
-    val IPv6SocketAddress: InetSocketAddress =
+    def IPv6SocketAddress: InetSocketAddress =
       InetSocketAddress.createUnresolved(IPv6Host, HttpPort)
```

Comment:
I had to make these `def`s so that the Scala.js linker can elide them. It's not ideal of course, but the alternative is splitting files. Reasonable compromise?